python/src/nanoarrow/_lib.pyx

```diff
@@ -474,20 +481,8 @@ cdef class Array:
         else:
             return None
 
-    def view(self):
```

Comment:
Is there a reason you are making this into a function instead of method? (for Schema it's still a method, so this gets a bit inconsistent)

Reply:
> Early on you made the suggestion that the `Array` and `ArrayView` were a little confusing, name wise. I wonder if renaming the `Array` to `CArray` (+ Schema and ArrayStream) would fix some of that. Then `ArrayView` could be renamed to `Array`, which would probably make more sense to everybody.

I was just writing up some thoughts on that! ;) -> https://github.com/apache/arrow-nanoarrow/issues/319